yt_dlp/extractor/fujitv.py

```diff
@@ -42,7 +51,10 @@ def _real_extract(self, url):
 
         return {
             'id': video_id,
-            'title': video_id,
+            'title': json_info.get('ep_title') or video_id,
```

Comment:
pukkandan made some changes recently to dlp core.
Title doesn't need to be fatal anymore so don't set it as is (dlp will give a more meaningful title now automatically if it's missing)
```suggestion
            'title': json_info.get('ep_title'),
```

Reply:
oh cool!